feature/platform/ate_tests/telemetry_inventory_test/telemetry_inventory_test.go

```diff
@@ -20,32 +20,29 @@ import (
 	"strings"
 	"testing"
 
+	"github.com/openconfig/featureprofiles/internal/components"
+	"github.com/openconfig/featureprofiles/internal/deviations"
 	"github.com/openconfig/featureprofiles/internal/fptest"
 	"github.com/openconfig/ondatra"
 	"github.com/openconfig/ondatra/gnmi"
+	"github.com/openconfig/ondatra/gnmi/oc"
 )
 
 var activeStatus string = "ACTIVE"
 
-var componentParent = map[string]string{
-	"fabric":      "Chassis",
-	"linecard":    "Chassis",
-	"powersupply": "Chassis",
-	"supervisor":  "Chassis",
-}
-
-var componentType = map[string]string{
-	"chassis":     "CHASSIS",
-	"fabric":      "FABRIC",
-	"fabricchip":  "INTEGRATED_CIRCUIT",
-	"linecard":    "LINECARD",
-	"fan":         "FAN",
-	"powersupply": "POWER_SUPPLY",
-	"supervisor":  "CONTROLLER_CARD",
-	"switchchip":  "INTEGRATED_CIRCUIT",
-	"transceiver": "TRANSCEIVER",
-	"tempsensor":  "SENSOR",
-}
+const (
```

Comment:
Let's drop all these constants and just inline the (longer oc constants). They create a double-maintenance burden and can add confusion. As one example, fabricChipType and switchChipType are both set to the same constant, which obscures the intention of the test.

Reply:
Done